src/bun.js/node/types.zig

```diff
@@ -4903,32 +4904,62 @@ pub const Process = struct {
             }
         }
 
-        var args = allocator.alloc(
-            JSC.ZigString,
-            // argv omits "bun" because it could be "bun run" or "bun" and it's kind of ambiguous
-            // argv also omits the script name
-            bun.argv.len -| 1,
-        ) catch bun.outOfMemory();
-        defer allocator.free(args);
-        var used: usize = 0;
-        const offset = 1;
+        var args = std.ArrayList(bun.String).initCapacity(temp_alloc, bun.argv.len - 1) catch bun.outOfMemory();
+        defer args.deinit();
+
+        var seen_run = false;
+        var prev: ?[]const u8 = null;
 
-        for (bun.argv[@min(bun.argv.len, offset)..]) |arg| {
-            if (arg.len == 0)
+        // we re-parse the process argv to extract execArgv, since this is a very uncommon operation
+        // it isn't worth doing this as a part of the CLI
+        for (bun.argv[1..]) |arg| {
```

Comment:
```suggestion
        for (bun.argv[@min(bun.argv.len, 1)..]) |arg| {
```

Reply:
is it possible to start the runtime with *zero* arguments?

Reply:
it should be possible, yes